features/car/classes.feature

```diff
@@ -119,6 +119,25 @@ Feature: Car - Mode flag
             | from | to | route       | turns                                      | classes                   |
             | a    | d  | ab,bc,cd,cd | depart,new name right,new name left,arrive | [()],[(tunnel)],[()],[()] |
 
+    Scenario: Car - We tag tunnel with a class
+        Background:
```

Comment:
Should we change the name of the scenario [above](https://github.com/Project-OSRM/osrm-backend/pull/4816/files#diff-4d1491a3addda1b6e121fd5f8961bebcR102)? Does it matter if we keep the same name for both scenarios?

Reply:
:+1: change to `Car - We tag classes without intersections`